network-api/networkapi/nav/tests/test_blocks.py

```diff
@@ -76,6 +77,39 @@ def test_default(self):
         self.assertEqual(block.url, url)
         self.assertTrue(block.is_external)
 
+    def test_valid_svg_upload(self):
+        """Test that an SVG file is accepted by the NavFeaturedItem model."""
+
+        # Create an image instance using the SVG format.
+        svg_image = ImageFactory(file__filename="icon.svg", file__extension="svg")
+
+        # Return a list of block data with the NavFeaturedItem factory, using the SVG image for "icon".
+        block_data = nav_factories.NavFeaturedItemFactory(icon=svg_image, external_url_link=True)
+
+        # Use the NavFeaturedItem model's clean() method to see if any validation errors are returned.
+        try:
+            nav_blocks.NavFeaturedItem().clean(block_data)
+        except StructBlockValidationError:
+            # We are expecting no errors. If one arrises, fail this test to let us know something is wrong.
+            self.fail("Clean method raised StructBlockValidationError unexpectedly")
```

Comment:
You can use `self.assertTrue(nav_blocks.NavFeaturedItem().clean(block_data))`. Since `.clean()` returns the `cleaned_data`, it will give back something, a "truthy" value when it goes well